src/java/com/unifina/signalpath/blockchain/ContractEventPoller.java

```diff
@@ -79,7 +86,7 @@ private void newFilter() {
 		List params = singletonList(singletonMap("address", contractAddress));
 		try {
 			rpc.rpcCall("eth_newFilter", params, ID_ADDFILTER);
-			keepPolling = true;
+			log.info("adding new filter");
```

Comment:
If we're gonna log, then let's include the `contractAddress` in the logged message.